test/e2e/README.md

```diff
@@ -192,6 +192,41 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on an existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been deployed, the following steps should be done for running the e2e tests.
```

Comment:
technically, you do not need to deploy Antrea to the cluster ahead of time, since the test cases will take care of it.